src/Combinatorics/SimplicialComplexes.jl

```diff
@@ -478,3 +478,27 @@ function Base.show(io::IO, K::SimplicialComplex)
     n = nvertices(K)
     print(io, "Abstract simplicial complex of dimension $(d) on $(n) vertices")
 end
+
+###############################################################################
+### Isomorphism
+###############################################################################
+
+@doc raw"""
+     is_isomorphic(K1::SimplicialComplex, K2::SimplicialComplex)
+
+Checks if the given simplicial complexes are isomorphic
+
+# Examples
+```jldoctest
+julia> K1 = simplicial_complex([[1,2,3],[2,3,4]]);
+
+julia> K2 = simplicial_complex([[1,2,3],[2,3,4]]);
+
+julia> is_isomorphic(K1, K2)
+true
+```
+"""
+function is_isomorphic(K1::SimplicialComplex, K2::SimplicialComplex)
+  return Polymake.topaz.isomorphic(pm_object(K1), pm_object(K2))
```

Comment:
```suggestion
  return Polymake.topaz.isomorphic(pm_object(K1), pm_object(K2))::Bool
```
Please always add return types for polymake functions.

Reply:
yup, sorry